fixtures/laserworld/el-400rgb-mk2.json

```diff
@@ -118,10 +118,273 @@
       }
     },
     "Color": {
-      "capability": {
-        "type": "ColorPreset",
-        "helpWanted": "What color(s) can be selected at which DMX value?"
-      }
+      "capabilities": [
+        {
+          "dmxRange": [0, 1],
+          "type": "Effect",
+          "effectName": "Color Macros",
+          "comment": "Multi Color",
+          "helpWanted": "Does this cycle all possible colours? Is this a colorFade or colorJump?"
+        },
+        {
+          "dmxRange": [2, 3],
+          "type": "ColorPreset",
+          "colors": ["#ff0000", "#00ff00", "#0000ff"],
+          "comment": "White"
```

Comment:
Shouldn't this be `#ffffff`? (Similar for other capabilities)

Based on the capability name, this laser might have a single beam with mixed colors. The description in https://github.com/OpenLightingProject/open-fixture-library/blob/master/docs/capability-types.md#property-colors is probably worded too strictly.

Reply:
According to the manual, it's an RGB Laser so it doesn't have a white beam. Perhaps I misunderstood the docs?

Reply:
> There are instances where the beam does look white

So at least the device *can* produce mixed colored beams. Let's use the simpler `#ffffff` etc. values then, those are probably more useful to users.

Reply:
No problem, just pushed that 👍